packages/editor/src/editor-styles/ast/stringify/identity.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable @wordpress/no-unused-vars-before-return */
```

Comment:
So this is the code which errors:
```js
Compiler.prototype.rule = function( node ) {
	const indent = this.indent();
	const decls = node.declarations;
	if ( ! decls.length ) {
		return '';
	}
```

Should we disable this check for objects somehow for the first iteration?

Reply:
Well, it's technically still wrong, but it raises an interesting point of whether there's mutation occurring in the `this.indent()` which needs to occur even if there's an early return. It seems like an edge case that we'd have (a) mutation, which (b) returns a value, and (c) needs to occur before the `return`.

Reply:
Yes, a lot of ifs here. It's hard to tell whether a given method mutates. Actually, this might be the case also in other places where you use regular function calls which modify some external variable.

Reply:
If it's obvious enough to a developer that they _need_ the mutation to occur, I'm perfectly fine with having one-off rule disabling (our "Disable reason" pattern).

Reply:
Thinking more about it, the rule is good, if someone wants to mutate, they can disable as you pointed out. 👍